sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -1263,6 +1264,8 @@ object SQLContext {
    */
   @transient private val instantiatedContext = new AtomicReference[SQLContext]()
 
+  @transient private val sqlListener = new AtomicReference[SQLListener]()
```

Comment:
Ah, I see that we need this in order to be able to return a value from `createListenerAndUI`.